fronts-client/src/types/Config.ts

```diff
@@ -20,14 +20,13 @@ interface Config {
   env: string;
   editions: string[];
   email: string;
-  avararUrl?: string;
+  avatarUrl?: string;
```

Comment:
@jonathonherbert Was it a typo in your commit?

Reply:
@mchv here's [the follow-up](https://github.com/guardian/facia-tool/pull/1430), with a test (now on PROD) if you're interested – tldr is that the link was missing a `https://` in the preview path, which preview did not appreciate.